app/models/aggregation.rb

```diff
@@ -14,8 +14,15 @@ class Aggregation < ActiveRecord::Base
   validate :aggregation, :workflow_version_present
 
   def self.scope_for(action, user, opts={})
-    return super unless (action == :show || action == :index) && !user.is_admin?
-    joins(:workflow).merge(Workflow.scope_for(:update, user, opts))
+    case
+    when action == :index && opts.has_key?(:public_workflow_ids)
```

Comment:
Just realised that effectively skips without valid`resource_ids`..i'll just override the index / show actions controlled_resources then. keep them out of the authorisation calls.